client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/content/get-domain-footer-details.ts

```diff
@@ -13,7 +13,9 @@ export default function getDomainFooterDetails( limit?: number ): ThankYouFooter
 			buttonText: translate( 'Learn the domain basics' ),
 			buttonHref: '/support/domains',
 			buttonOnClick: () => {
-				recordTracksEvent( 'calypso_thank_you_footer-domain-essentials' );
+				recordTracksEvent( 'calypso_thank_you_footer_link_click', {
+					type: 'footer-domain-essentials',
+				} );
```

Comment:
```suggestion
				recordTracksEvent( 'calypso_thank_you_footer_link_click', {
					context: context, // 'domain-only' or 'transfer-to-user'
					type: 'domain-essentials',
				} );
```
Since we already have `footer` in the event name, I think mentioning it in the type prop might be superfluous.

I wonder if we shouldn't pass a `context` prop with this event as well. While it's easy enough to keep track of the fact that `domain-essentials` belongs to the domain congrats page, I'm not sure this will hold true for all footer links on all congrats pages.

Here in `getDomainFooterDetails`, which is used in both `DomainTransferToAnyUser` and `DomainOnlyThankYou`, we should consider using a new argument passed to `getDomainFooterDetails` for the `context` prop.

Side note: is there a convention around dashes vs underscores for Tracks event properties..?

Reply:
>I wonder if we shouldn't pass a `context` prop with this event as well. 

Indeed the same footer blurb may be shown in different Congrats pages. It'd be useful to pass a `context` prop to identify where the link came from. e.g. `plan-only`, `domain-only`, etc.

>is there a convention around dashes vs underscores for Tracks event properties..?

From what I've seen, tracks event names and property _names_ use `_`. I'm not aware of conventions for property _values_, but here's an example that uses `-` for reporting property values:
https://github.com/Automattic/wp-calypso/blob/fbbd52ec657736dfb967e6d29b6f493bcc4d0e36/apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js#L24-L27